src/pyggdrasil/tree_inference/_interface.py

```diff
@@ -32,3 +32,12 @@
     2  # Homozygous mutation observed. See Eq. (8) on p. 5 of the SCITE paper.
 )
 MISSING_ENTRY: int = 3  # Missing entry.
+
+# Cell Attachment Vector
+# entries refer to mutations/nodes with root 0
+# indices counted from 0 refer to cell(sample numbers
+Cell_Attachment_Vector = Array
+
+# Ancestor Matrix
+# as in SCITE paper without last row trunicated
+Ancestor_Matrix = Array
```

Comment:
Added descriptive, yet trivial datatypes here. They are only used in tree_inference._simulate but I need these types for the tests to pass certain function inputs. Is this a good style?

Reply:
Yes, I think it makes sense! :slightly_smiling_face: 

Just a small stylistic comment: in Python we use `snake_case` (function names) or `CamelCase` (classes). I recommend looking from time to time at [Google Style Guide](https://google.github.io/styleguide/pyguide.html#s3.16.2-naming-conventions), which is a great compendium on style.